data/json/mapgen/house18.json

```diff
@@ -0,0 +1,152 @@
+                ".|G      |-|-|      G|..",
+                ".2E      |L|L|      E2..",
+                ".|EEAF   |+|+|    >AE|..",
+                ".|-----|   |   |-----|..",
+                ".| 7>7 |   |   |77   |..",
+                ".|     +   |   +    :|..",
+                ".2:    |-+-|-+-|   @@2..",
+                ".2:    |Sl8|8lS|   @@2..",
+                ".|     |tl8|8lt|    :|..",
+                ".| @@  |---|---|  B  |..",
+                ".|:@@6 +  9|9  + CCC |..",
+                ".|-22--|---|---|--22-|.."
+            ],
+            "terrain": {
+                ".": "t_grass",
+                " ": "t_floor",
+                "|": "t_wall",
+                "-": "t_wall",
+                "+": "t_door_c",
+                "2": "t_window_domestic",
+                "3": "t_door_locked",
+                "l": "t_linoleum_gray",
+                "s": "t_sidewalk",
+                "t": "t_linoleum_gray",
+                "6": "t_floor",
+                "7": "t_floor",
+                "8": "t_linoleum_gray",
+                "9": "t_floor",
+                ":": "t_floor",
+                ">": "t_floor",
+                "?": "t_floor",
+                "@": "t_floor",
+                "^": "t_floor",
+                "A": "t_floor",
+                "B": "t_floor",
+                "C": "t_floor",
+                "E": "t_floor",
+                "F": "t_floor",
+                "G": "t_floor",
+                "S": "t_linoleum_gray",
+                "L": "t_floor",
+                "W": "t_floor"
+            },
+            "furniture": {
+                "6": "f_table",
+                "7": "f_bookcase",
+                "8": "f_bathtub",
+                "9": "f_rack",
+                ":": "f_dresser",
+                ">": "f_counter",
+                "?": "f_sofa",
+                "@": "f_bed",
+                "^": "f_indoor_plant",
+                "A": "f_sink",
+                "B": "f_chair",
+                "C": "f_desk",
+                "E": "f_cupboard",
+                "F": "f_fridge",
+                "G": "f_oven",
+                "S": "f_sink",
+                "L": "f_locker",
+                "W": "f_washer"
+            },
+            "toilets": {
+                "t": { }
+            },
+            "comment": "specific terrain/furniture/radiation. doing this until grass_or_dirt is supported ---v",
+            "set": [
+                { "point": "terrain", "id": "t_dirt", "x": 0, "y": [ 1, 23 ], "repeat": [ 5, 10 ] },
+                { "point": "terrain", "id": "t_dirt", "x": [ 22, 23 ], "y": [ 1, 23 ], "repeat": [ 10, 20 ] }
+            ],
+            "comment": "specific things ---v",
+            "add": [
+                { "item": "television", "x": 4, "y": 2, "chance": 55 },
+                { "item": "television", "x": 4, "y": 16, "chance": 20 }
+            ],
+            "comment": "randomly picked from a group ---v",
+            "place_items": [
```

Comment:
It would be great to see item spawn scale proportionally influence `repeat`.
For now I will play around with numbers in-game and see what works and what doesn't.